src/libcmd/include/nix/command.hh

```diff
@@ -0,0 +1,2 @@
+#warning "Including nix/command.hh is deprecated. Include nix/cmd/command.hh instead."
```

Comment:
Would it make sense to generate these files on-the-fly rather than checking them out?

(they aren't big nor anything, but I imagine them being a bit confusing and very annoying when browsing the codebase with a fuzzy-finder)

Reply:
I personally don't think these files are worth the trouble. It's not too hard to do the bulk rename with a small script to see which library it was moved to.

Reply:
As a maintainer of a package that uses the headers headers, I will have to add a version check conditional around my imports anyway.

Automating this guidance is probably not worth the effort for our limited ecosystem of library consumers.

Reply:
If that's the consensus, I am happy to remove the shims. @roberth do you see any way we can make the transition easier for downstream consumers?

I think the worst part is figuring out where each include file went, and the shims addressed this by mentioning the location of the new file. I could make a awk script to migrate to the new locations but it won't help if you need to support both.